src/API/OnboardingTasks.php

```diff
@@ -610,4 +641,244 @@ public function get_status() {
 
 		return rest_ensure_response( $status );
 	}
+
+	/**
+	 * Get the onboarding tasks.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function get_tasks() {
+		$profiler_data         = get_option( Onboarding::PROFILE_DATA_OPTION, array() );
+		$installed_plugins     = PluginsHelper::get_installed_plugin_slugs();
+		$product_types         = $profiler_data['product_types'];
```

Comment:
I just noticed that a fresh site may throw an error on this line since the default for `PROFILE_DATA_OPTION` is an empty array:

![image](https://user-images.githubusercontent.com/444632/129979416-403acbaa-c8f5-4aa5-85fc-fa6bbc6cec17.png)

Reply:
Good catch!  Updated in https://github.com/woocommerce/woocommerce-admin/pull/7512/commits/c181247c20465b4fe2b5199d7af9c27f0571c0e4